lib/tasks/export.rake

```diff
@@ -56,7 +56,7 @@ module ExportTask
       # Export Conjur database
       FileUtils.mkpath out_dir.join('backup')
       dbdump = out_dir.join('backup/conjur.db')
-      call(%(pg_dump -Fc -f \"#{dbdump}\" #{ENV['DATABASE_URL']})) ||
+      call(%(pg_dump --exclude-table="messages" -Fc -f \"#{dbdump}\" #{ENV['DATABASE_URL']})) ||
```

Comment:
We should extract "messages" to variable instead of hardcoded in string
and add comment to why we are doing so

Reply:
I agree that we need to add a comment.
since this is the only place that we used messages I don't think we need to export this to variable